examples/job_crud.py

```diff
@@ -20,6 +20,8 @@
 
 import yaml
 
+from time import sleep
```

Comment:
the [pycodestyle test](https://travis-ci.org/github/kubernetes-client/python/jobs/770682844) is picky about the import. Could you make it 
```
from os import path
from time import sleep
 
import yaml

from kubernetes import client, config
```
?

Reply:
@roycaihw I have made the above mentioned change. Please verify now. Thanks

Reply:
@roycaihw Looks like the build is failing again. Can you please tell me what's happening?

Reply:
are there any special character on the line after " from time import sleep"?

==============================================
diff --git a/examples/job_crud.py b/examples/job_crud.py
index 2174ef7..ab02761 100644
--- a/examples/job_crud.py
+++ b/examples/job_crud.py
@@ -18,7 +18,7 @@ Creates, updates, and deletes a job object.
 
 from os import path
 from time import sleep
- 
+
 import yaml
 
 from kubernetes import client, config

Reply:
https://travis-ci.org/github/kubernetes-client/python/jobs/770874568 has the diff you can check out

Reply:
No there is no special characters except newline @yliaog

Reply:
@yliaog looks like there was an space along with the newline and i fixed it now. Please check and approve once the build is successful